src/library/Collections.nim

```diff
@@ -47,21 +47,21 @@ proc defineSymbols*() =
     #  labels: library, enhancement
 
     builtin "append",
-        alias       = doubleplus, 
-        rule        = InfixPrecedence,
+        alias = doubleplus,
```

Comment:
```suggestion
        alias       = doubleplus, 
```

Also, if you could restore the indentation in the built-in method headers, it would really appreciated! 😉  (This and all the other cases...)

Reply:
> Also, if you could restore the indentation in the built-in method headers, it would really appreciated! 😉 (This and all the other cases...)

I'll do it! 😉